doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -555,6 +555,14 @@ Libraries for NFC
 
 |no_changes_yet_note|
 
+  * :file:`ncs/nrf/subsys/nfc/lib/platform.c`:
```

Comment:
We do not refer to specific files in the release notes. If the changes are seen in a library, refer to it.